package.json

```diff
@@ -63,20 +52,23 @@
     "regenerator-runtime": "^0.11.0",
     "resolve": "^1.4.0",
     "rimraf": "^2.6.2",
-    "rollup": "^0.56.2",
-    "rollup-plugin-babel": "^3.0.2",
-    "rollup-plugin-commonjs": "^8.2.1",
+    "rollup": "^0.65.2",
+    "rollup-plugin-babel": "^4.0.0",
+    "rollup-plugin-commonjs": "^9.1.6",
     "rollup-plugin-flow": "^1.1.1",
-    "rollup-plugin-json": "^2.1.1",
+    "rollup-plugin-json": "^3.0.0",
     "rollup-plugin-node-builtins": "^2.1.1",
-    "rollup-plugin-node-globals": "^1.1.0",
-    "rollup-plugin-node-resolve": "^3.0.0",
+    "rollup-plugin-node-globals": "^1.3.0",
+    "rollup-plugin-node-resolve": "^3.4.0",
     "slash": "^1.0.0",
     "string-length": "^2.0.0",
     "strip-ansi": "^4.0.0",
     "typescript": "^2.2.2",
     "watchify": "^3.9.0"
   },
+  "resolutions": {
```

Comment:
ugh, is this really needed?

Reply:
Most likely not, was experimenting a bit with it since some deps in the tree still use babel-core.